### ISSUE TITLE
How to install plugins?

### ISSUE THREAD

**Issue by dalareo** [bitnami/bitnami-docker-discourse#28]
**Description**

Unable to install plugins

**Steps to reproduce the issue:**

1. Enter /bitnami/discourse/plugins folder
2. Execute git clone plugin_git_url
3. Restart app
4. Get error: "We're sorry, but something went wrong. The issue has been logged for investigation. Please try again later."

**Additional environment details (AWS, VirtualBox, Docker for MAC, physical, etc.):**

Running inside Rancher

**Comment by carrodher** [comment]
Hi,

I have successfully installed some plugins running:
```
cd /opt/bitnami/discourse/
RAILS_ENV=production bundle exec rake plugin:install repo=plugin_git_url
RAILS_ENV=production bundle exec rake assets:precompile
```

The procedure according to the official documentation is more similar to the steps you describe. Can you do the following?

1. Try my workaround. 
2. What command did you execute in step "3. Restart app"? I would like to reproduce exactly your steps.

**Comment by dalareo** [comment]
My container is restarted after I execute
```
RAILS_ENV=production bundle exec rake plugin:install repo=https://github.com/jonmbake/discourse-ldap-auth
```

By "restarting app" I mean restart container.

**Comment by dbarranco** [comment]
Hi @dalareo 

Could you please explain a bit more the issue you are facing?

I used the command you shared to install the LDAP auth plugin and it worked:

```
cd /opt/bitnami/discourse
RAILS_ENV=production bundle exec rake plugin:install repo=https://github.com/jonmbake/discourse-ldap-auth
RAILS_ENV=production bundle exec rake assets:precompile
```

And the precompilation of the assets worked as expected:

```
Start compiling CSS: 2017-10-20 07:51:28 UTC
Compiling css for default 2017-10-20 07:51:28 UTC
precompile target: desktop Dark
precompile target: mobile Dark
precompile target: desktop_rtl Dark
precompile target: mobile_rtl Dark
precompile target: desktop Default
precompile target: mobile Default
precompile target: desktop_rtl Default
precompile target: mobile_rtl Default
precompile target: desktop
precompile target: mobile
precompile target: desktop_rtl
precompile target: mobile_rtl
Done compiling CSS: 2017-10-20 07:51:34 UTC
```

**Comment by dalareo** [comment]
It works!!

**Comment by cdupont** [comment]
Installing plugins is not clear for me as well.
I deployed discourse using your docker images and mounted the volumes as per the README file of this repo. 
I saw this doc: https://meta.discourse.org/t/install-plugins-in-discourse/19157
But where should I download the plugin? On the host mounted volumes? How can I run this command:
```
 ./launcher rebuild app
```
In the container? Should I rebuild the image?
Thanks!!
